syncmaster/db/migrations/versions/2023-11-23_0004_create_connection_table.py

```diff
@@ -30,6 +30,7 @@ def upgrade():
         "connection",
         sa.Column("id", sa.BigInteger(), nullable=False),
         sa.Column("group_id", sa.BigInteger(), nullable=False),
+        sa.Column("type", sa.String(length=128), nullable=False),
```

Comment:
nit: actually, 128 chars is too long, I think 16 or 23 will be enough

Reply:
The migrations unit test is failing due to type differences between SQLAlchemy and PostgreSQL. However, setting the default length to 255 resolves the issue, and the test passes

Reply:
You can pass different `ChoiseType(..., impl=StringType(length=...))`, according to https://sqlalchemy-utils.readthedocs.io/en/stable/data_types.html#sqlalchemy_utils.types.choice.ChoiceType